src/sonic-yang-models/yang-models/sonic-vxlan.yang

```diff
@@ -0,0 +1,212 @@
+module sonic-vxlan {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-vxlan";
+    prefix svxlan;
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+    import sonic-vlan {
+        prefix svlan;
+    }
+	import sonic-types {
+		prefix stypes;
+	}
+
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC VXLAN";
+
+    revision 2021-04-12 {
+        description
+            "First revision.";
+    }
+
+    container sonic-vxlan {
+
+        container VXLAN_TUNNEL {
+
+            description "config db VXLAN_TUNNEL table";
+
+            list VXLAN_TUNNEL_LIST {
+
+                key "name";
+                max-elements 1;
+
+                leaf name {
+                    must "(starts-with(../name, 'vtep'))"
```

Comment:
Please use the string pattern to enforce this.

Reply:
Why do we need a list if the number of entries itself is one?

Reply:
Removed this restriction of having to start with vtep.